src/lib/command_line.ml

```diff
@@ -105,11 +113,13 @@ let client_term =
         let actions = [
           "describe", `Describe;
           "status", `Status;
+          "list", `List;
+          "kill", `Kill;
         ] in
         required
         & pos 0 (some (enum actions)) None
         & info [] ~doc:"Action to do on the current cluster:\
```

Comment:
`on the current cluster` does not really apply (copied from the other subcommand? ;) )